### PR TITLE
Add Copyright and authors (EBrains SGA3)

### PR DESCRIPTION
**Pull request by antonelepfl**

### CODE REVIEW

**Comment by antonelepfl** [comment]
@ASVG @alex4200 

This is for EBrains (the new wizard that we are developing)
Could you please check if everything is correct?

File: README.md [hunk 1]
```diff
@@ -27,11 +27,18 @@ npm run build
...
-And then when we are fine with the changes, we should PR to https://github.com/lbologna/ebrains-sim-cl-dev
+This wizard will be embeded into [Cellular Level Simulation](https://github.com/ebrains-cls-interactive/ebrains-cls-interactive.github.io) page
+
+
+## Funding & Acknowledgment
+- This project/research was supported by funding to the Blue Brain Project, a research center of the École polytechnique fédérale de Lausanne (EPFL), from the Swiss government’s ETH Board of the Swiss Federal Institutes of Technology.
+- This project/research has received funding from the European Union’s Horizon 2020 Framework Programme for Research and Innovation under the Specific Grant Agreement No. 945539 (Human Brain Project SGA3).
+- This project/research received funding from the European Union’s Horizon 2020 Framework Programme for Research and Innovation under the Framework Partnership Agreement No. 650003 (HBP FPA).
+
+Copyright (c) 2021 - 2021 Blue Brain Project/EPFL
```
Comment (alex4200) [inline README.md:44]: Just say "Copyright (c) 2021 Blue Brain Project/EPFL"

**Review by alex4200 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ASVG) [reply README.md]: I am bit confused... should it also be Copyright (c) 2017-2021 Blue Brain Project/EPFL?

**Comment by antonelepfl** [comment]
I have modified the statements leaving only the SGA3
Akiko, this is the new one that we are developing for EBrains (in this branch) the old one is in a branch called `collab` with a PR

**Comment by alex4200** [comment]
LGTM

**Pull request merged**